lib/compat/wordpress-6.2/script-loader.php

```diff
@@ -191,3 +191,7 @@ function gutenberg_enqueue_global_styles_custom_css() {
 	}
 }
 add_action( 'wp_enqueue_scripts', 'gutenberg_enqueue_global_styles_custom_css' );
+
+
+remove_action( 'wp_body_open', 'wp_global_styles_render_svg_filters' );
+remove_action( 'in_admin_header', 'wp_global_styles_render_svg_filters' );
```

Comment:
This removes the call that adds the SVGs for the duotone presets to the body of the page.